docs/core/whats-new/dotnet-9/libraries.md

```diff
@@ -527,11 +528,14 @@ The following code shows how to set the option (the `Book` type definition is sh
 
 :::code language="csharp" source="../snippets/dotnet-9/csharp/Serialization.cs" id="RespectNullable":::
 
-You can also enable this setting globally using the `System.Text.Json.JsonSerializerOptions.RespectNullableAnnotations` feature switch in your project file (for example, _.csproj_ file):
+> [!NOTE]
```

Comment:
Not sure if this markdown form is valid here, but didn't know how to best render a note section.